inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -883,5 +884,404 @@ std::shared_ptr<CNNNetworkImpl> convertFunctionToICNNNetwork(const std::shared_p
 
     return cnnNetworkImpl;
 }
+
+    std::shared_ptr<CNNNetworkImpl> convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function>& graph, const ICNNNetwork &network) {
```

Comment:
Why do you duplicate the code? It is not correct! Please use one implementation of `convertFunctionToICNNNetwork` for both interfaces.

Reply:
Is it possible to change existing functions signature?

Reply:
@alexeytarakanov  yes, adding new parameter with default value does not break source compatibility.

Reply:
First of all in this code you can call `convertFunctionToICNNNetwork()` with bool flag inside the function without bool flag.

Reply:
Done in new commit. Parameter bool keep_constant_inputs has default value - false.